app/Livewire/GlobalSearch.php

```
@@ -62,7 +105,144 @@ public static function clearTeamCache($teamId)
+        if ($query === 'new' || str_starts_with($query, 'new ')) {
+            $this->isCreateMode = true;
+            $this->loadCreatableItems();
+
+            // Check for sub-commands like "new project", "new server", etc.
+            $detectedType = $this->detectSpecificResource($query);
+            if ($detectedType) {
+                $this->navigateToResource($detectedType);
+            } else {
+                // If no specific resource detected, reset selection state
+                $this->cancelResourceSelection();
+            }
```

PHPMD (2.15.0)

The method updatedSearchQuery uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)